packages/ses/src/commons.js

```diff
@@ -191,6 +193,10 @@ export const stringEndsWith = uncurryThis(stringPrototype.endsWith);
 export const stringIncludes = uncurryThis(stringPrototype.includes);
 export const stringIndexOf = uncurryThis(stringPrototype.indexOf);
 export const stringMatch = uncurryThis(stringPrototype.match);
+//
+export const iteratorNext = uncurryThis(iteratorPrototype.next);
+export const iteratorThrow = uncurryThis(iteratorPrototype.throw);
```

Comment:
Might be more accurate to call these `generator*` since they do not work on arbitrary iterators.

Reply:
I considered `generatorIterable*`, `generatedIterable*` or `generatee`, which would be more precise but weirder